### ISSUE TITLE
There is overlap at the top of the labels with the notches on the MAD marker.

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-variability#266]
From https://github.com/phetsims/center-and-variability/issues/231#issuecomment-1593278444

> The font sizes and the placement of the distances look great. There is overlap at the top of the labels with the notches on the MAD marker.

![Image](https://github.com/phetsims/center-and-variability/assets/109630002/50bd988e-9ce3-4e28-aeaf-fce2bf57ec89)

> 
> Let's move the notches up so the distance measurement at the top doesn't overlap with the MAD marker. Looking really good!

**Comment by samreid** [comment]
![image](https://github.com/phetsims/center-and-variability/assets/679486/30f6fdf7-5dc2-4fea-aff2-344e45017cff)

Fixed in the commit and ready for design review.

**Comment by catherinecarter** [comment]
The mean vertical line should be extended vertically, and the rectangle should follow. The rectangle should be at the top of the vertical line, and the first distance line slightly lower than the height of the vertical bar. The non-overlapping MAD distance indicator looks good.

**Comment by samreid** [comment]
@matthew-blackman implemented this yesterday in https://github.com/phetsims/center-and-variability/issues/281. @catherinecarter can you please test and close if all is well?

**Comment by catherinecarter** [comment]
I checked it out, and there are a couple minor tweaks. 
- the height of the rectangle looks great
- the notches above the rectangle look great
- the distance markers look great

In this screenshot: 
<img width="1000" alt="image" src="https://github.com/phetsims/center-and-variability/assets/109630002/4a18b0b0-ae4e-4914-857c-9764f8448188">

I noticed the data points are z-ordered behind the horizontal lines (correct), but distances are behind the data points. The distance is difficult to read.

Also, it appears as though the length of the equation could use some of the horizontal space. The numerator is squished when it looks like there's room to the right.
